pow-migration/src/genesis/types.rs

```diff
@@ -39,21 +39,22 @@ pub enum Error {
 ///
 /// ```text
 ///     1                  2                  3                  4         5
-/// --- | ---------------- | ---------------- | ---------------- | ------- |
+/// --- | ---------------- | ---------------- | ---------------- | ------- | ----
 ///
 /// ```
 ///
 /// 1. Validator registration window start block.
 /// 2. Validator registration window end block. This block is also the pre-stake
 ///    registration window start.
-/// 3. Pre-stake registration window end block. This block is also the activation
-///    window start.
-/// 4. The final block in the PoW chain that will be taken as genesis block for the
-///    PoS chain. This block must have a block number that can be an election block
-///    number in the PoS chain.
+/// 3. Pre-stake registration window end block.
```

Comment:
Point 2 needs to be updated, since the validator registration end block does not necessary equals the pre-stake start

Reply:
In this code, sadly yes. I'll change this in my upcoming set of changes